src/components/[guild]/Requirements/RequirementErrorConfigContext.tsx

```diff
@@ -5,7 +5,7 @@ type RequirementErrorConfig = Partial<Record<RequirementType, string>>
 
 const REQ_ERROR_CONFIG: RequirementErrorConfig = {
   GALAXY: "Galxe API error, please try again later",
-  TWITTER: "X API error, please try again later",
+  // TWITTER: "X API error, please try again later",
```

Comment:
Why has this been commented? Typing-related, or we don't need it?

Reply:
Typing-related & we don't really need it anymore. :D Will remove it in another PR!